tensorflow/core/ops/math_ops.cc

```diff
@@ -969,7 +969,11 @@ REGISTER_OP("ArgMax")
     .Attr("T: numbertype")
     .Attr("Tidx: {int32, int64} = DT_INT32")
     .Attr("output_type: {int32, int64} = DT_INT64")
-    .SetShapeFn(ArgOpShape);
+    .SetShapeFn(ArgOpShape)
+	.Doc(R"doc(Returns the index with the largest value across axes of a tensor.
+inputs: Must all be the same size and shape.
```

Comment:
Let's remove the `inputs: Must all..` lines, both here and below in `ArgMin`. These are no longer arguments to the op and having them here causes the builds to crash.

Reply:
Done